### DISCUSSION TITLE
Metaphor Story - Redirect to Mobile Pages with useDetectMobile in React | Metaphore (SCP)

### DISCUSSION THREAD

**Discussion by giscus** [StreetCommunityProgrammer/metaphore#41] [Metaphor - Discussion]
# Metaphor Story - Redirect to Mobile Pages with useDetectMobile in React | Metaphore (SCP)

darkterminal telling a story about Redirect to Mobile Pages with useDetectMobile in React

https://metaphore.vercel.app/stories/javascript/redirect-to-mobile-pages-with-usedetectmobile-in-react

**Comment by mkubdev** [comment]
This one is super cool! ⚡

**Reply by darkterminal** [reply]
Also nested comment or reply thread.... 🔥

**Comment by darkterminal** [comment]
Amazing platform btw... we can comments and always on Github.
